<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>Ku Lab - Publications</title>
  <meta name="description" content="Ku Lab -- Publications.">
  <link rel="stylesheet" href="http://localhost:4000/Kao_Lab/css/main.css">
  <link rel="canonical" href="http://localhost:4000/Kao_Lab/publications/">
  <link rel="shortcut icon" type ="image/x-icon" href="http://localhost:4000/Kao_Lab/images/IPMB_logo.png">
  <script src="https://kit.fontawesome.com/05beb1a375.js" crossorigin="anonymous"></script>



<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-153761675-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-153761675-1');
</script>


</head>


  <body>

    <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container-fluid">

  <div class="navbar-header">
	  <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse-1" aria-expanded="false">
		<span class="sr-only">Toggle navigation</span>
		<span class="icon-bar"></span>
		<span class="icon-bar"></span>
		<span class="icon-bar"></span>
	  </button>

    <a class="navbar-brand" href="http://localhost:4000/Kao_Lab/">Ku Lab</a>
	</div>

	<div class="collapse navbar-collapse" id="navbar-collapse-1">
	  <ul class="nav navbar-nav navbar-right">
		<li><a href="http://localhost:4000/Kao_Lab/team">Members</a></li>
    <li><a href="http://localhost:4000/Kao_Lab/about">About</a></li>
<!---  <li><a href="http://localhost:4000/Kao_Lab/research">Research</a></li>  --->
    <li><a href="http://localhost:4000/Kao_Lab/publications">Publications</a></li>
    <li><a href="http://localhost:4000/Kao_Lab/pictures">Gallery</a></li>
		<li><a href="http://localhost:4000/Kao_Lab/positions">Join us!</a></li>
	  </ul>
	</div>

  </div>
</div>


    <div class="container-fluid">
      <div class="row">
        <div id="gridid" class="col-sm-12">
  <h1 id="publications">Publications</h1>

<h4><a href="https://scholar.google.com/citations?user=1BDzxq8AAAAJ&amp;hl=en">Google Scholar</a><br /></h4>
<h4><a href="https://www.webofscience.com/wos/author/record/1835861">Web of Science</a><br /></h4>
<p><br /></p>

<h3 id="selected-works-by-chuan-ku-and-others">Selected works by Chuan Ku and others</h3>

<div class="row">

  <div class="col-sm-6 clearfix">
    <div class="well">
      <pubtit>Widespread distribution and evolution of poxviral entry-fusion complex proteins in giant viruses</pubtit>
      <p><img src="http://localhost:4000/Kao_Lab/images/pubpic/MicrobiolSpectrum2023.Figure5B.png" class="img-responsive" width="33%" style="float: left" /></p>
      <p>The origin and early evolution of giant viruses as revealed by the membrane fusion proteins essential for poxvirus cell entry</p>
      <p><em>Sheng Kao, Chi-Fei Kao, Wen Chang, Chuan Ku</em></p>
      <p><strong><a href="https://journals.asm.org/doi/10.1128/spectrum.04944-22">Microbiology Spectrum (2023)</a></strong></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
    </div>
  </div>

  <div class="col-sm-6 clearfix">
    <div class="well">
      <pubtit>Single-cell transcriptomics unveils xylem cell development and evolution</pubtit>
      <p><img src="http://localhost:4000/Kao_Lab/images/pubpic/GenomBiology2023.Figure5A.png" class="img-responsive" width="33%" style="float: left" /></p>
      <p>A single-cell view of development in xylem, the most abundant tissue on Earth, across divergent tree species, incl. Trochodendron, a vessel-less angiosperm native to Taiwan.</p>
      <p><em>Chia-Chun Tung, Shang-Che Kuo, Chia-Ling Yang, Jhong-He Yu, Chia-En Huang, Pin-Chien Liou, Ying-Hsuan Sun, Peng Shuai, Jung-Chen Su, Chuan Ku, Ying-Chung Jimmy Lin</em></p>
      <p><strong><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02845-1">Genome Biology (2023)</a></strong></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
    </div>
  </div>

</div>

<div class="row">

  <div class="col-sm-6 clearfix">
    <div class="well">
      <pubtit>Rampant nuclear-mitochondrial-plastid phylogenomic discordance in globally distributed calcifying microalgae</pubtit>
      <p><img src="http://localhost:4000/Kao_Lab/images/pubpic/NewPhytologist2022.Figure_6C.website.png" class="img-responsive" width="33%" style="float: left" /></p>
      <p>Extensive cytonuclear and inter-organellar phylogenomic incongruence illuminate the complex evolutionary history and inheritance modes of genomes in unicellular algae</p>
      <p><em>Tzu-Tong Kao, Tzu-Haw Wang, Chuan Ku</em></p>
      <p><strong><a href="https://nph.onlinelibrary.wiley.com/doi/10.1111/nph.18219">New Phytologist (2022)</a></strong></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
    </div>
  </div>

  <div class="col-sm-6 clearfix">
    <div class="well">
      <pubtit>Unraveling gene content variation across eukaryotic giant viruses based on network analyses and host associations</pubtit>
      <p><img src="http://localhost:4000/Kao_Lab/images/pubpic/VirusEvol2021.Figure_2.website.png" class="img-responsive" width="33%" style="float: left" /></p>
      <p>Gene sharing patterns across giant virus families often contradict with the core-gene phylogeny, but they can be partially explained by associations with different eukaryotic hosts</p>
      <p><em>Tsu-Wang Sun, Chuan Ku</em></p>
      <p><strong><a href="https://doi.org/10.1093/ve/veab081">Virus Evolution (2021)</a></strong></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
    </div>
  </div>

</div>

<div class="row">

  <div class="col-sm-6 clearfix">
    <div class="well">
      <pubtit>Giant virus-eukaryote interactions as ecological and evolutionary driving forces</pubtit>
      <p><img src="http://localhost:4000/Kao_Lab/images/pubpic/mSystems2021.Figure_2.website.png" class="img-responsive" width="33%" style="float: left" /></p>
      <p>Giant eukaryotic viruses and their infection processes in ecological and evolutionary contexts</p>
      <p><em>Chuan Ku</em></p>
      <p><strong><a href="https://doi.org/10.1128/mSystems.00737-21">mSystems (2021)</a></strong></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
    </div>
  </div>

  <div class="col-sm-6 clearfix">
    <div class="well">
      <pubtit>Host range and coding potential of eukaryotic giant viruses</pubtit>
      <p><img src="http://localhost:4000/Kao_Lab/images/pubpic/GirusEukCophylo.png" class="img-responsive" width="33%" style="float: left" /></p>
      <p>Diversity of eukaryotic hosts and virus-encoded gene functions, and their evolution across giant virus lineages and families</p>
      <p><em>Tsu-Wang Sun, Chia-Ling Yang, Tzu-Tong Kao, Tzu-Haw Wang, Ming-Wei Lai, Chuan Ku</em></p>
      <p><strong><a href="https://www.mdpi.com/1999-4915/12/11/1337">Viruses (2020)</a></strong></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
    </div>
  </div>

</div>

<div class="row">

  <div class="col-sm-6 clearfix">
    <div class="well">
      <pubtit>A single-cell view on alga-virus interactions reveals sequential transcriptional programs and infection states</pubtit>
      <p><img src="http://localhost:4000/Kao_Lab/images/pubpic/InfGradient.png" class="img-responsive" width="33%" style="float: left" /></p>
      <p>Dissecting giant virus infection one cell at a time</p>
      <p><em>Chuan Ku, Uri Sheyn, Arnau Sebé-Pedrós, Shifra Ben-Dor, Daniella Schatz, Amos Tanay, Shilo Rosenwasser, Assaf Vardi</em></p>
      <p><strong><a href="https://advances.sciencemag.org/content/6/21/eaba4137">Science Advances (2020)</a></strong></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
    </div>
  </div>

  <div class="col-sm-6 clearfix">
    <div class="well">
      <pubtit>Using single-cell transcriptomics to understand functional states and interactions in microbial eukaryotes</pubtit>
      <p><img src="http://localhost:4000/Kao_Lab/images/pubpic/PhilTransB_Fig1.png" class="img-responsive" width="33%" style="float: left" /></p>
      <p>An overview of current scRNA-seq methods and their potential applications for microbial eukaryotes</p>
      <p><em>Chuan Ku, Arnau Sebé-Pedrós</em></p>
      <p><strong><a href="https://royalsocietypublishing.org/doi/10.1098/rstb.2019.0098">Philosophical Transactions B (2019)</a></strong></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
    </div>
  </div>

</div>

<div class="row">

  <div class="col-sm-6 clearfix">
    <div class="well">
      <pubtit>A natural barrier to lateral gene transfer from prokaryotes to eukaryotes revealed from genomes: the 70% rule</pubtit>
      <p><img src="http://localhost:4000/Kao_Lab/images/pubpic/PhyloDissection.png" class="img-responsive" width="33%" style="float: left" /></p>
      <p>Phylogenomic dissection of gene origins across prokaryotic groups and eukaryotes</p>
      <p><em>Chuan Ku, William F. Martin</em></p>
      <p><strong><a href="https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-016-0315-9">BMC Biology (2016)</a></strong></p>
      <p class="text-danger"><strong> </strong></p>
      <p> Recommended by <a href="http://f1000.com/prime/726862983">Faculty of 1000</a></p>
    </div>
  </div>

  <div class="col-sm-6 clearfix">
    <div class="well">
      <pubtit>Endosymbiotic origin and differential loss of eukaryotic genes</pubtit>
      <p><img src="http://localhost:4000/Kao_Lab/images/pubpic/EPCs.png" class="img-responsive" width="33%" style="float: left" /></p>
      <p>Testing the contributions of endosymbioses and lateral gene transfers to the eukaryotic gene repertoire</p>
      <p><em>Chuan Ku, Shijulal Nelson-Sathi, Mayo Roettger, Filipa L. Sousa, Peter J. Lockhart, David Bryant, Einat Hazkani-Covo, James O. McInerney, Giddy Landan, William F. Martin</em></p>
      <p><strong><a href="http://www.nature.com/doifinder/10.1038/nature14963">Nature (2015)</a></strong></p>
      <p class="text-danger"><strong> </strong></p>
      <p> Recommended by <a href="http://f1000.com/prime/725729035">Faculty of 1000</a></p>
    </div>
  </div>

</div>

<div class="row">

  <div class="col-sm-6 clearfix">
    <div class="well">
      <pubtit>Endosymbiotic gene transfer from prokaryotic pangenomes: Inherited chimerism in eukaryotes</pubtit>
      <p><img src="http://localhost:4000/Kao_Lab/images/pubpic/TreeInterpretation.png" class="img-responsive" width="33%" style="float: left" /></p>
      <p>Eukaryotic genome evolution in the context of prokaryotic evolution: pangenomes, endosymbiosis, sex, recombinations and tree interpretations</p>
      <p><em>Chuan Ku, Shijulal Nelson-Sathi, Mayo Roettger, Sriram Garg, Einat Hazkani-Covo, William F. Martin</em></p>
      <p><strong><a href="https://www.pnas.org/content/112/33/10139">Proceedings of the National Academy of Sciences of the United States of America (2015)</a></strong></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
    </div>
  </div>

</div>

<p>   </p>

<h3 id="full-list-of-publications-by-chuan-ku-and-others">Full list of publications by Chuan Ku and others</h3>

<p>Merit of integrating in situ transcriptomics and anatomical information for cell annotation and lineage construction in single‑cell analyses of Populus <br />
  <em>Ying-Lan Chen, Jo-Wei Allison Hsieh, Shang-Che Kuo, Chung-Ting Kao, Chia-Chun Tung, Jhong-He Yu, Tien-Hsien Chang, Chuan Ku, Jianbo Xie, Deqiang Zhang, Quanzi Li, Ying-Chung Jimmy Lin </em><br /><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-024-03227-5">Genome Biology (2024)</a></p>

<p>Daily turnover of active giant virus infection during algal blooms revealed by single-cell transcriptomics <br />
  <em>Gur Hevroni, Flora J. Vincent, Chuan Ku, Uri Sheyn, Assaf Vardi </em><br /><a href="https://www.science.org/doi/10.1126/sciadv.aba4137">Science Advances (2023)</a></p>

<p>Widespread distribution and evolution of poxviral entry-fusion complex proteins in giant viruses <br />
  <em>Sheng Kao, Chi-Fei Kao, Wen Chang, Chuan Ku </em><br /><a href="https://journals.asm.org/doi/10.1128/spectrum.04944-22">Microbiology Spectrum (2023)</a></p>

<p>Single-cell transcriptomics unveils xylem cell development and evolution <br />
  <em>Chia-Chun Tung, Shang-Che Kuo, Chia-Ling Yang, Jhong-He Yu, Chia-En Huang, Pin-Chien Liou, Ying-Hsuan Sun, Peng Shuai, Jung-Chen Su, Chuan Ku, Ying-Chung Jimmy Lin </em><br /><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02845-1">Genome Biology (2023)</a></p>

<p>Widespread distribution and evolution of poxviral entry-fusion complex proteins in giant viruses <br />
  <em>Sheng Kao, Chi-Fei Kao, Wen Chang, Chuan Ku </em><br /><a href="https://www.researchgate.net/publication/367286701_Widespread_distribution_and_evolution_of_poxviral_entry-fusion_complex_proteins_in_giant_viruses">bioRxiv (2023)</a></p>

<p>Complete genome sequence of <i>Phaeobacter</i> sp. strain G2 isolated from lab culture of the calcifying alga <i>Emiliania huxleyi</i> <br />
  <em>Siwat Chang, Tzu-Haw Wang, Ming-Wei Lai, Chuan Ku </em><br /><a href="https://journals.asm.org/doi/10.1128/mra.01067-22">Microbiology Resource Announcements (2023)</a></p>

<p>Daily turnover of active giant virus infection during algal blooms revealed by single-cell transcriptomics <br />
  <em>Gur Hevroni, Flora J. Vincent, Chuan Ku, Uri Sheyn, Assaf Vardi </em><br /><a href="https://www.biorxiv.org/content/10.1101/2022.10.15.512338v1">bioRxiv (2022)</a></p>

<p>Rampant nuclear-mitochondrial-plastid phylogenomic discordance in globally distributed calcifying microalgae <br />
  <em>Tzu-Tong Kao, Tzu-Haw Wang, Chuan Ku </em><br /><a href="https://nph.onlinelibrary.wiley.com/doi/10.1111/nph.18219">New Phytologist (2022)</a></p>

<p>Single-cell transcriptomics unveils xylem cell development and evolution <br />
  <em>Chia-Chun Tung, Shang-Che Kuo, Chia-Ling Yang, Chia-En Huang, Jhong-He Yu, Ying-Hsuan Sun, Peng Shuai, Jung-Chen Su, Chuan Ku, Ying-Chung Jimmy Lin </em><br /><a href="https://www.biorxiv.org/content/10.1101/2021.10.16.464627v1">bioRxiv (2021)</a></p>

<p>Unraveling gene content variation across eukaryotic giant viruses based on network analyses and host associations <br />
  <em>Tsu-Wang Sun, Chuan Ku </em><br /><a href="https://doi.org/10.1093/ve/veab081">Virus Evolution (2021)</a></p>

<p>Giant virus-eukaryote interactions as ecological and evolutionary driving forces <br />
  <em>Chuan Ku </em><br /><a href="https://doi.org/10.1128/mSystems.00737-21">mSystems (2021)</a></p>

<p>Host range and coding potential of eukaryotic giant viruses <br />
  <em>Tsu-Wang Sun, Chia-Ling Yang, Tzu-Tong Kao, Tzu-Haw Wang, Ming-Wei Lai, Chuan Ku </em><br /><a href="https://www.mdpi.com/1999-4915/12/11/1337">Viruses (2020)</a></p>

<p>A single-cell view on alga-virus interactions reveals sequential transcriptional programs and infection states <br />
  <em>Chuan Ku, Uri Sheyn, Arnau Sebé-Pedrós, Shifra Ben-Dor, Daniella Schatz, Amos Tanay, Shilo Rosenwasser, Assaf Vardi </em><br /><a href="https://advances.sciencemag.org/content/6/21/eaba4137">Science Advances (2020)</a></p>

<p>Did giant and large dsDNA viruses originate before their eukaryotic hosts? <br />
  <em>Chuan Ku, Tsu-Wang Sun </em><br /><a href="https://www.pnas.org/content/117/6/2747">Proceedings of the National Academy of Sciences of the United States of America (2020)</a></p>

<p>Using single-cell transcriptomics to understand functional states and interactions in microbial eukaryotes <br />
  <em>Chuan Ku, Arnau Sebé-Pedrós </em><br /><a href="https://royalsocietypublishing.org/doi/10.1098/rstb.2019.0098">Philosophical Transactions B (2019)</a></p>

<p>Tracking infection dynamics at single-cell level reveals highly-resolved expression programs of a large virus infecting algal blooms <br />
  <em>Chuan Ku, Uri Sheyn, Arnau Sebé-Pedrós, Shifra Ben-Dor, Daniella Schatz, Amos Tanay, Shilo Rosenwasser, Assaf Vardi </em><br /><a href="https://www.biorxiv.org/content/10.1101/757542v1">bioRxiv (2019)</a></p>

<p>Complete genome sequence of <i>Sulfitobacter</i> sp. strain D7, a virulent bacterium isolated from an Emiliania huxleyi algal bloom in the North Atlantic <br />
  <em>Chuan Ku, Noa Barak-Gavish, Mark Maienschein-Cline, Stefan J. Green, Assaf Vardi </em><br /><a href="https://mra.asm.org/content/7/19/e01379-18">Microbiology Resource Announcements (2018)</a></p>

<p>Bacterial virulence against an oceanic bloom-forming phytoplankter is mediated by algal DMSP <br />
  <em>Noa Barak-Gavish, Miguel Frada, Chuan Ku, Peter A. Lee, Giacomo R. DiTullio, Sergey Malitsky, Asaph Aharoni, Stefan J. Green, Ron Rotkopf, Elena Kartvelishvily, Uri Sheyn, Daniella Schatz, Assaf Vardi </em><br /><a href="https://advances.sciencemag.org/content/4/10/eaau5716">Science Advances (2018)</a></p>

<p>Late mitochondrial origin is an artefact <br />
  <em>William F. Martin, Mayo Roettger, Chuan Ku, Sriram G. Garg, Shijulal Nelson-Sathi, Giddy Landan </em><br /><a href="https://academic.oup.com/gbe/article/9/2/373/2986798">Genome Biology and Evolution (2017)</a></p>

<p>A natural barrier to lateral gene transfer from prokaryotes to eukaryotes revealed from genomes: the 70% rule <br />
  <em>Chuan Ku, William F. Martin </em><br /><a href="https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-016-0315-9">BMC Biology (2016)</a></p>

<p>Endosymbiotic origin and differential loss of eukaryotic genes <br />
  <em>Chuan Ku, Shijulal Nelson-Sathi, Mayo Roettger, Filipa L. Sousa, Peter J. Lockhart, David Bryant, Einat Hazkani-Covo, James O. McInerney, Giddy Landan, William F. Martin </em><br /><a href="http://www.nature.com/doifinder/10.1038/nature14963">Nature (2015)</a></p>

<p>Endosymbiotic gene transfer from prokaryotic pangenomes: Inherited chimerism in eukaryotes <br />
  <em>Chuan Ku, Shijulal Nelson-Sathi, Mayo Roettger, Sriram Garg, Einat Hazkani-Covo, William F. Martin </em><br /><a href="https://www.pnas.org/content/112/33/10139">Proceedings of the National Academy of Sciences of the United States of America (2015)</a></p>

<p>Plastid origin: Who, when and why? <br />
  <em>Chuan Ku, Mayo Roettger, Verena Zimorski, Shijulal Nelson-Sathi, Filipa L. Sousa, William F. Martin </em><br /><a href="https://pbsociety.org.pl/journals/index.php/asbp/article/view/asbp.2014.045/0">Acta Societatis Botanicorum Poloniae (2014)</a></p>

<p>Endosymbiotic theory for organelle origins <br />
  <em>Verena Zimorski, Chuan Ku, William F. Martin, Sven B. Gould </em><br /><a href="https://www.sciencedirect.com/science/article/pii/S1369527414001283">Current Opinion in Microbiology (2014)</a></p>

<p>Molecular evolution of the actin-like MreB protein gene family in wall-less bacteria <br />
  <em>Chuan Ku, Wen-Sui Lo, Chih-Horng Kuo </em><br /><a href="https://www.sciencedirect.com/science/article/pii/S0006291X14004732?via%3Dihub">Biochemical and Biophysical Research Communications (2014)</a></p>

<p>Molecular evolution of the substrate utilization strategies and putative virulence factors in mosquito-associated <i>Spiroplasma</i> species <br />
  <em>Tean-Hsu Chang, Wen-Sui Lo, Chuan Ku, Ling-Ling Chen, Chih-Horng Kuo </em><br /><a href="https://academic.oup.com/gbe/article/6/3/500/576835">Genome Biology and Evolution (2014)</a></p>

<p>Complete genome sequence of <i>Spiroplasma apis</i> B31T (ATCC 33834), a bacterium associated with May disease of honeybees (<i>Apis mellifera</i>) <br />
  <em>Chuan Ku, Wen-Sui Lo, Ling-Ling Chen, Chih-Horng </em><br /><a href="https://mra.asm.org/content/2/1/e01151-13">Genome Announcements (2014)</a></p>

<p>Phylogenetic and cophylogenetic analyses of the leaf-nodule symbiosis in <i>Ardisia</i> subgenus <i>Crispardisia</i> (Myrsinaceae): Evidence from nuclear and chloroplast markers and bacterial <i>rrn</i> operons <br />
  <em>Chuan Ku, Jer-Ming Hu </em><br /><a href="https://www.journals.uchicago.edu/doi/abs/10.1086/673306">International Journal of Plant Sciences (2014)</a></p>

<p>Horizontal transfer of potential mobile units in phytoplasmas <br />
  <em>Chuan Ku, Wen-Sui Lo, Chih-Horng Kuo </em><br /><a href="https://www.tandfonline.com/doi/full/10.4161/mge.26145">Mobile Genetic Elements (2013)</a></p>

<p>Comparison of metabolic capacities and inference of gene content evolution in mosquito-associated <i>Spiroplasma diminutum</i> and <i>S. taiwanense</i> <br />
  <em>Wen-Sui Lo, Chuan Ku, Ling-Ling Chen, Tean-Hsu Chang, Chih-Horng Kuo </em><br /><a href="https://academic.oup.com/gbe/article/5/8/1512/617412">Genome Biology and Evolution (2013)</a></p>

<p>The complete plastid genome sequence of Madagascar periwinkle <i>Catharanthus roseus</i> (L.) G. Don: Plastid genome evolution, molecular marker identification, and phylogenetic implications in asterids <br />
  <em>Chuan Ku, Wan-Chia Chung, Ling-Ling Chen, Chih-Horng Kuo </em><br /><a href="https://journals.plos.org/plosone/article/authors?id=10.1371/journal.pone.0068518">PLoS ONE (2013)</a></p>

<p>Complete genomes of two dipteran-associated spiroplasmas provided insights into the origin, dynamics, and impacts of viral invasion in <i>Spiroplasma</i> <br />
  <em>Chuan Ku, Wen-Sui Lo, Ling-Ling Chen, Chih-Horng Kuo </em><br /><a href="https://academic.oup.com/gbe/article-lookup/doi/10.1093/gbe/evt084">Genome Biology and Evolution (2013)</a></p>

<p>Complete plastid genome sequence of the basal asterid <i>Ardisia polysticta</i> Miq. and comparative analyses of asterid plastid genomes <br />
  <em>Chuan Ku, Jer-Ming Hu, Chih-Horng Kuo </em><br /><a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0062548">PLoS ONE (2013)</a></p>

<p>Dispersal of airborne pollen in Chatienshan nature reserve, northern Taiwan, with emphasis on Taiwan beech <br />
  <em>Chuan Ku, Liang-Chi Wang, Su-Hwa Chen </em><br /><a href="http://tai2.ntu.edu.tw/taiwania/abstract.php?type=abstract&amp;id=1196">Taiwania (2012)</a></p>


</div>

      </div>
    </div>

    <head>
  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css">
</head>

<div id="footer" class="panel">
  <div class="panel-footer">
	<div class="container-fluid">
	  <div class="row">


		<div class="col-sm-4">
      Funding:
      <figure class="fourth">
        <a href="https://www.sinica.edu.tw/en"><img src="http://localhost:4000/Kao_Lab/images/logopic/AS logo-1.png" style="width: 50px"></a>
        <a href="https://www.embo.org/"><img src="http://localhost:4000/Kao_Lab/images/logopic/EMBO_logo1.png" style="width: 220px"></a>
        <br/>
        <a href="https://ipmb.sinica.edu.tw/en"><img src="http://localhost:4000/Kao_Lab/images/logopic/IPMB_logo2.png" style="width: 300px"></a>
        <br/>
        <a href="https://www.nstc.gov.tw/?l=en"><img src="http://localhost:4000/Kao_Lab/images/logopic/NSTC_logo1.png" style="width: 300px"></a>
      </figure>
      <br/>
      <p>&copy 2019 Ku Lab. Site made with <a href="https://jekyllrb.com">Jekyll</a> adapted from <a href='https://www.allanlab.org/aboutwebsite.html'>Milan Allan template</a>.</a></p>
		</div>


		<div class="col-sm-4">
		  R220/R222<br />
      Institute of Plant and Microbial Biology<br />
      Academia Sinica<br />
		  128, Sec. 2, Academia Rd., Taipei 115201<br />
		  Taiwan <br />
      (<a href="https://www.ifs.sinica.edu.tw/posts/139244">Directions</a>)<br/>
      <br/>
      <a href="mailto:kulabMEEE@gmail.com" title="Email us">Lab email</a><br/>
      +886-2-27871147 (lab/offices)<br/>
      +886-2-27871059 (PI office)<br/>
      <br/>
      
      <div id="social-media">
          
          
              
              
          
              
              
                  <a href="https://www.twitter.com/chuanku_" title="Twitter"><i class="fab fa-twitter fa-3x"></i></a>
              
          
              
              
                  <a href="https://orcid.org/0000-0001-6414-4423" title="ORCID"><i class="fab fa-orcid fa-3x"></i></a>
              
          
              
              
                  <a href="https://scholar.google.com/citations?hl=en&user=1BDzxq8AAAAJ&view_op=list_works" title="Google Scholar"><i class="fab fa-google-scholar fa-3x"></i></a>
              
          
              
              
                  <a href="https://github.com/chuanku-lab" title="GitHub"><i class="fab fa-github fa-3x"></i></a>
              
          
      </div>
      

      <div style="text-align: center;">
      <img src="http://localhost:4000/Kao_Lab/images/logopic/Kulab_logo.png" style="width: 250px">
      </div>

		</div>

    <div class="col-sm-4">
      <div class="gmap_canvas"><iframe width="300" height="300" id="gmap_canvas" src="https://maps.google.com/maps?q=Institute%20of%20Plant%20and%20Microbial%20Biology%20(IPMB)&t=&z=13&ie=UTF8&iwloc=&output=embed" frameborder="0" scrolling="no" marginheight="0" marginwidth="0"></iframe><a href="https://www.whatismyip-address.com"></a></div><style>.mapouter{position:relative;text-align:right;height:500px;width:600px;}.gmap_canvas {overflow:hidden;background:none!important;height:350px;width:500px;max-width:105%}</style>
    </div>

	  </div>
	</div>
  </div>
</div>

<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
<script src="http://localhost:4000/Kao_Lab/js/bootstrap.min.js"></script>


  </body>

</html>
